src/DateRangePicker.jsx

```diff
@@ -19,8 +19,6 @@ const PureRenderMixin = React.addons.PureRenderMixin;
 const absoluteMinimum = moment(new Date(-8640000000000000 / 2)).startOf('day');
 const absoluteMaximum = moment(new Date(8640000000000000 / 2)).startOf('day');
 
-React.initializeTouchEvents(true);
```

Comment:
Can you not check whether the  `initializeTouchEvents` method exists on React and then call it if it does? That way we can continue to support older versions of React.